### DISCUSSION TITLE
Collecting Graphics of Integral / Spiral Dynamics Stages

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#963] [Research]
This is a thread to collect graphical representations of integral / spiral dynamic stages and model.

## AQAL: An Integral Map - Spiral Dynamics Stages

Source [Formless Mountain](http://www.formlessmountain.com/integral_gallery_store.htm) by Steve Self

![image](https://github.com/life-itself/community/assets/180658/0d7fc6fd-92e6-4ad1-a7a8-03f3b33ec645)

**Comment by matthewmccarthy11** [comment]
@asimong you had a nice one in the chat today

**Reply by asimong** [reply]
yes... though actually I don't recall where it comes from. I will see if I can find the source

**Reply by asimong** [reply]
![Wilber-levels+Lines1](https://github.com/life-itself/community/assets/2691463/e1e64d87-d383-40f8-b993-c0243c5d96dd)

**Reply by asimong** [reply]
Similar ones can be found in many places on the web, and this is an example of a relatively full one.

**Comment by asimong** [comment]
![comparison-table](https://github.com/life-itself/community/assets/2691463/c0af90f5-9eba-4d96-a6ac-61dc0db4ee19)

**Reply by asimong** [reply]
Correlating even more schemes, though (sorry) I don't recall where this came from. Fairly obviously, it is upside down compared with the other one. Seems more or less consistent with the other chart.

**Reply by rufuspollock** [reply]
Wilber has a cross match of schemes at the end of the Integral Psychology. However i don't think there is any mapping to "colors" (and i personally think that is a bit dubious in any case)

**Comment by asimong** [comment]
A point that I would like to see expressed here – so I will express it now! – is that all these schemes seem to be based on similar assumptions and cultural background. The fairly well diffused critique by Nora Bateson (pithily at https://twitter.com/NoraBateson/status/1425142926254956550 ) has a point, while seeming to throw the baby rather angrily and unreflectively out with the bathwater.  I can't muster the enthusiasm to follow up e.g. https://www.youtube.com/watch?v=8cMnPZ2MJiU or indeed to subsidise Bateson's rants by paying to get past a paywall. I'm wondering also whether it is worth following up on https://mysticalmobius.net/2021/10/03/stage-theory-is-bs-pt-5/

But the point I would concur with is that Bateson is pointing out, to my mind correctly, the cultural dependency of these stage models. They seem to me to chart the natural human responses to the kind of complexity faced in contemporary Western culture, but not necessarily in other cultures. I go back to George Kelly and Person Construct Psychology for an approach that seems to me much sounder and less biased. Thus, societies with different complexities are liable to have different stages of development. What Kelly's theory is pointing out is that people typically don't fluidly change their paradigms or worldviews continuously, but only in a series of what we could see as "crises", where the disconfirming evidence has built up too much to be ignored.

**Comment by matthewmccarthy11** [comment]
@asimong in the twitter thread you mentioned up there I found a diagram to the question I asked in #959, so I will be posting it here. I don't know where it's originally from, and based on how it looks it might potentially cause some headaches. Curious if you have any initial reactions
![group-stage-theory](https://github.com/life-itself/community/assets/131390186/ee085e3b-d56e-4b65-a2a2-f5334b81b07b)

**Reply by asimong** [reply]
Well, @matthewmccarthy11  as I said in #959 — a bit more flesh: putting a single dividing line between "interior" and "exterior" – and indeed between "individual" and "collective" – seems to me now not only unhelpfully simplistic, but also covertly individualistic. Just "me" and "the world outside me". No, no, and thrice no! And, by the way, I'm not leaving out self-criticism here, I have gone down the same (blind alley?) path back in 2009. 

However, that was only addressing the bare "four quadrants" before examining the content. Having looked at the content, one related thing I notice is that many of the supposedly "exterior–individual" things seem to me essentially relational: to do with individuals in relationship, not just with the supposedly isolated individual. Then, looking at the "exterior–collective" it seems to me full of similar themes. So is there a real division between individual and collective in the exterior domain? Maybe there is a sense that what is labelled "exterior" has a somewhat larger scope than those labelled "interior", but I see no clear dividing line. 
If I were to rearrange the chart, maybe I would put only things that are to do with the public behaviour of individuals in the individual–exterior box. 

Next … (oh, there is so much to comment on here…) looking at the top half, the "individual", what is the division between "what I experience" and "what I do"??? I sense an unhelpful dualism here, unless we are much clearer in setting out a more obvious criterion for separating these.

Back to the overall feel of the chart …
The "levels" do make quite some sense, in terms of the assumptions and culture in which we sit. Two things stand out for me.
1. There seems to be a very clear "red is bad" through to "teal is best", similar to what I have already pointed out critically with the "pathways to liberation" materials.  https://docs.google.com/spreadsheets/d/1KPIvCD_XIGThKEsqZ-qKQwY0Op5pSe_r7uqksetW5FQ/edit#gid=10 Sure, we would no doubt like to live and work in "teal" structures and organisations, but is that the right level at all times? Referring back to the state of the world this past year, with apparently more rather than less war, is a teal organisation the best one for an actual battlefield? There's a timing issue here. It takes a while to engage not only the frontal lobes, but also to cross-check and harmonise between hemispheres, and when immediate action is needed, that's not a good survival strategy. I guess we can think of other examples.
2. The teal levels seem very fuzzy. Yes, they all use words (shibboleths?) which we all agree with, often as an unreflected and uncritical reflex, but what do they actually mean, in any terms where we can reasonably distinguish whether the criteria are met or not? If we don't have such clarity, the obvious danger is that we get taken up in virtue signalling. Not so good.

This is just a quick reply — do point out where I may have been over-hasty in response; and otherwise I'd like to continue the dialogue.
